server/stores.go

```diff
@@ -47,25 +47,37 @@ func hasRoleContext(ctx context.Context) (string, bool) {
 	}
 }
 
-type superAdminKey string
+type userKey string
```

Comment:
would prefer to name this `userContextKey` for clarity and consistency with `serverContextKey`

Reply:
yeah good call.